lib/kubeclient/watch_stream.rb

```diff
@@ -1,54 +1,63 @@
 require 'json'
-require 'net/http'
+require 'http'
 module Kubeclient
   module Common
     # HTTP Stream used to watch changes on entities
     class WatchStream
       def initialize(uri, http_options, format: :json)
         @uri = uri
-        @http = nil
-        @http_options = http_options.merge(read_timeout: nil)
+        @http_client = nil
+        @http_options = http_options
```

Comment:
@msufa you removed the `read_timeout` but apparently you haven't introduced anything similar for the new `http` gem, don't we need something like:

http://www.rubydoc.info/gems/http/0.9.8/HTTP/Timeout/Global